app/controllers/facilitator/projects_controller.rb

```diff
@@ -44,11 +44,11 @@ def find_facilitator
   end
 
   def project_creation_params
-    params.require(:project_creation_form).permit(project_attributes: [:name, :location, :start_on, students_attributes: [:id, :name, :class_name, :birthday, :comments, :_destroy]])
+    params.require(:project_creation_form).permit(project_attributes: [:name, :location, :start_on, :end_on, :status, students_attributes: [:id, :name, :class_name, :birthday, :comments, :_destroy]])
```

Comment:
Is there a good reason for adding :end_on and :status right now into the strong params? At the moment, these fields aren't exposed in the form view.